nilearn/_utils/cache_mixin.py

```diff
@@ -108,11 +108,11 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
     memory: instance of joblib.Memory or string
         Used to cache the function call.
 
-    func_memory_level: int, optional
+    func_memory_level: int
```

Comment:
> In CacheMixin, make memory levels mandatory for function *_cache*. It is logic and they are never used without them.

I talk about function `_cache`, for which `func_memory_level` is now a mandatory parameter. `cache` is another function and can be called without them.